test/oscar/compiler/AND.oscar.out

```diff
@@ -0,0 +1,4 @@
+true 
+false 
+false 
+false 
```

Comment:
This is going to be annoying, but ... the output files that are attached to tests which use `Println` (like this one) might need newlines added at the end.

This is simply b/c `Println` appends a newline to the end of its output, which means that even though the output file content is correct, it won't find the last newline and fail the test.

We should test this to make sure before doing work, but that's my feel on this.